src/api/views/main.py

```diff
@@ -1,38 +1,47 @@
 from flask import Blueprint, session
 from flask import request, render_template, redirect, url_for
-from api.scrapper.news_get import extract, ALLOWED_URLS
+from api.core import logger, create_response
+from api.scrapper import collect_articles
+from api.repository.mongo import save_articles
+import json
+
+
+def get_error_msg_if_exists():
```

Comment:
misleading function name, it not only gets data but also wipes it out from session

Reply:
removed